objectivec/GPBUtilities.h

```diff
@@ -188,9 +381,7 @@ CF_EXTERN_C_END
 
 //%PDDM-DEFINE GPB_ACCESSORS()
 //%
-//%//
-//%// Get/Set the given field of a message.
-//%//
+//%/** @name Get/Set a given field from/to a message. */
```

Comment:
Likely doesn't need to change since there is nothing api wise for this to anchor too.